app/controllers/stories_controller.rb

```diff
@@ -48,6 +48,10 @@ def show
 
   private
 
+  def assign_hero_banner
+    @hero_display_ad = DisplayAd.for_display(area: "home_hero", user_signed_in: false)
```

Comment:
@benhalpern, @jaw6 I was wondering if you could provide some clarification on this: 

> These types of billboards are not able to target logged_in users for now, but we don't need to codify this as a validation.

I stand to be corrected but I think we're talking about omitting targeting on a logged_in user through the audience segments and not specifically about the user_signed_in attribute on a display_ad. 

My assumption is that we'd still want these billboards to be able to show up appropriately based on whether a display ad is for a signed in or signed out user, but we would not target the signed in user with audience segments for now. Hence my initial reaction is that this should change as follows: 

```suggestion
    @hero_display_ad = DisplayAd.for_display(area: "home_hero", user_signed_in: user_signed_in?)
```
Again, I don't feel certain and could use with soem clarity.